app/assets/stylesheets/_buttons.scss

```diff
@@ -1,5 +1,5 @@
+.add-to-vocab-btn,
 .add_vocab_button {
```

Comment:
oh sorry one tiiinny last thing. It looks like the add_vocab_button class isn't actually used anywhere anymore since deleting that helper method, so you can get rid of the class here now and just leave these styles assigned to the add-to-vocab-btn class instead.